src/controllers/csi/gc/unmounted.go

```diff
@@ -0,0 +1,76 @@
+package csigc
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/spf13/afero"
+)
+
+const (
+	defaultMaxUnmountedCsiVolumeAge = 14 * 24 * time.Hour
+	MaxUnmountedCsiVolumeAgeEnv     = "MAX_UNMOUNTED_VOLUME_AGE"
```

Comment:
doesn't appear to be used in another package
```suggestion
	maxUnmountedCsiVolumeAgeEnv     = "MAX_UNMOUNTED_VOLUME_AGE"
```